chapter_14_simple_form.asciidoc

```diff
@@ -165,6 +164,7 @@ And then:
 ----
 ====
 // RITA: Is this note missing "for some help" at the end?
+// CSANAD: Not sure what you mean?
```

Comment:
@ritaf-ORM could you please clarify?

Reply:
In the previous edition of the book, this note box ended with "Check out django-crispy-forms
and django-floppyforms for some help." You removed the mention of django-floppyforms, which is fine. I'm asking if you accidentally removed "for some help" to complete the sentence.